### DISCUSSION TITLE
Github's language

### DISCUSSION THREAD

**Discussion by xyuhjh2007** [community/community#33926] [Other Feature Feedback, Questions, & Ideas] [answered]
Why Github only has got English?

**Comment by james-villanueva-freee** [comment]
Hi. Now that I saw your question, I am wondering too. 
But this might give you some insight https://github.blog/2010-07-13-github-in-your-language/#:~:text=Just%20click%20on%20one%20of,to%20switch%20to%20that%20language.

**Comment by Kirtishukla2004** [comment, marked as answer]
Today, thanks to github , github is  turning on a number of languages that you can view GitHub in other than English.
Currently  i think github  have Japanese,French,Serbian,Deutsch(german ),Svenska(Swedish), Hrvatski(Croatian), Polski (Polish) and Nederlands (Dutch) etc...
You will be able to see links to these languages in the footer of each GitHub page. Just click on one of the languages to switch to it – it will store your language choice in your session. If your browser has the Accept-Language header set and you have not selected a language choice

**Reply by softorangetech1122** [reply]
It says it has been deleted -- https://github.blog/2010-07-13-github-in-your-language/#:~:text=July%2013%2C%202010-,Edit%20(11/18/2016)%3A%20GitHub%20no%20longer%20supports%20languages%20other%20than%20English
